### ISSUE TITLE
Tags being displayed in table field

### ISSUE THREAD

**Issue by underdog** [grafana/grafana#37151]
**What happened**:
table visualization is displaying all group by tag values in addition to alias value

**What you expected to happen**:
field should just display alias value

**How to reproduce it (as minimally and precisely as possible)**:
create a table based on data, add multiple group by tag fields, alias it with $tag_<some group tag>

**Anything else we need to know?**:
N/A

**Environment**:
- Grafana version: Grafana v8.1.0-beta2 (147704deb9)
- Data source type & version: InfluxDB version: 1.8.6 
- OS Grafana is installed on: Linux raspberrypi 5.10.17-v7l+ # 1421 SMP Thu May 27 14:00:13 BST 2021 armv7l GNU/Linux 
- User OS & Browser: Windows 10 - Chrome, Brave, Edge
- Grafana plugins: Standard Table Visualization Plugin
- Others:
![image](https://user-images.githubusercontent.com/510960/126773153-9c406929-3787-420b-a379-da396691373f.png)

**Comment by kylebrandt** [comment]
Can you also screenshot / share what the Transforms are?

**Comment by underdog** [comment]
attaching transforms screenshot
![image](https://user-images.githubusercontent.com/510960/126787435-79724553-4833-4e4f-a87a-fa260b8ecbce.png)

**Comment by torkelo** [comment]
This is a side effect of https://github.com/grafana/grafana/pull/36702 that added tags the response, which changes the naming of a field. 

Since influxdb has internal naming / alias logic it needs to set the title property on the old legacy TimeSeries object, 
https://github.com/grafana/grafana/blob/main/public/app/plugins/datasource/influxdb/influx_series.ts#L55

If title is set Grafana when translating this to DataFrame & fieldConfig will set displayNameFromDS which will sidestep Grafana's built-in naming 
https://github.com/grafana/grafana/blob/main/packages/grafana-data/src/dataframe/processDataFrame.ts#L98

**Comment by gabor** [comment]
@torkelo just to clarify, you mean we set into `timeseries.title` the same thing as we set into `timeseries.target`? 
do we do it always? or only when an `alias` is specified?

in other words, is this the diff?
```diff
diff --git a/public/app/plugins/datasource/influxdb/influx_series.ts b/public/app/plugins/datasource/influxdb/influx_series.ts
index 48cbc7bdd8..1c7fc7fcfb 100644
--- a/public/app/plugins/datasource/influxdb/influx_series.ts
+++ b/public/app/plugins/datasource/influxdb/influx_series.ts
@@ -52,6 +52,7 @@ export default class InfluxSeries {
         }

         output.push({
+          title: seriesName,
           target: seriesName,
           datapoints: datapoints,
           tags: series.tags,
```

thanks!

(NOTE: there was another change since then, influxdb now directly returns a dataframe, but that's not important, i will adjust the code as needed)

**Comment by torkelo** [comment]
yes, that diff should fix it, we do the same in Prometheus (or did before they switched to DataFrame)

**Comment by MaestroGlanz** [comment]
I encountered a similar problem in #37179 . But in my case without transform.

@gabor @torkelo  As I'm new to grafana: Can one of you shortly explain/give futher reading, what this diff does? If I understand this correctly, this is a git diff of the grafana source code. Do I have to compile it locally myself then, or will this be fixed in an upcoming version?

**Comment by gabor** [comment]
hi @MaestroGlanz , no need to compile anything, this will be fixed in an upcoming version (should be fixed in version 8.1 (non-beta)).
